p2p/simulations/http.go

```diff
@@ -320,7 +320,8 @@ func (s *Server) StartMocker(w http.ResponseWriter, req *http.Request) {
 	mockerid := req.Context().Value("mock").(string)
 
 	if len(s.Mockers) == 0 {
-		http.Error(w, "mocker not configured", http.StatusInternalServerError)
+		//don't require a mocker to be present
+		s.JSON(w, http.StatusNotModified, "No mocker configured")
```

Comment:
@lmars @holisticode lets make a deal. if you buiild an external mocker that generattes the events corresponding to a mocker we get rid of it.
it could be part of p2psim or a js lib spitting out node stop / start events sampling a ontime / offtime probability distribution